pkg/kubelet/cm/memorymanager/memory_manager.go

```diff
@@ -239,6 +241,8 @@ func (m *manager) AddContainer(pod *v1.Pod, container *v1.Container, containerID
 
 // GetMemoryNUMANodes provides NUMA nodes that used to allocate the container memory
 func (m *manager) GetMemoryNUMANodes(pod *v1.Pod, container *v1.Container) sets.Set[int] {
+	logger := klog.LoggerWithValues(klog.Background(), "pod", klog.KObj(pod), "containerName", container.Name)
```

Comment:
The same here, please use `klog.TODO()` and explain in a comment when it should be replaced with proper context.

Reply:
Will explore if we can pass context into this function.